css/project-list-pages.styl

```diff
@@ -43,6 +43,10 @@
     .pill-button
       margin-right: 10px
       padding: 1em
+      color: rgb(64,64,64)
+      font-weight: normal
+      border-color: rgb(64,64,64) !important
+      box-shadow: rgb(64, 64, 64) 0px 0px 1px 0px inset !important
```

Comment:
See my comment above. I've figured it out. The markup for the top pagination buttons is in the wrong place, and inheriting grey colours from its parent.